src/common/replication_common.h

```diff
@@ -106,9 +106,6 @@ class replication_options
     int32_t fd_grace_seconds;
 
     int32_t log_private_file_size_mb;
-    int32_t log_private_batch_buffer_kb;
-    int32_t log_private_batch_buffer_count;
-    int32_t log_private_batch_buffer_flush_interval_ms;
```

Comment:
I notice that this pull request remove `log_private_batch_buffer` related options for private log. Could you please explain why they are unused?

Reply:
To keep consistency with shared log